src/components/SearchBar/__tests__/SearchBar.spec.js

```diff
@@ -68,6 +68,17 @@ describe('SearchBar', () => {
     expect(input.value).toBe('');
   });
 
+  it('does not show the clear button if there is no search term present', async () => {
```

Comment:
nit we could consolidate this with the first test 'has the x button disabled when no text was entered'

Reply:
good catch - I removed this outdated test

Reply:
and you could also check that it is disabled in this test

Reply:
But it's not disabled